gtsfm/utils/features.py

```diff
@@ -47,3 +47,65 @@ def normalize_coordinates(coordinates: np.ndarray, intrinsics: Cal3Bundler) -> n
     """
 
     return np.vstack([intrinsics.calibrate(x[:2].reshape(2, 1)) for x in coordinates])
+
```

Comment:
is features.py the right name for this file?

Reply:
keypoint_utils.py?

Reply:
Good point. But should I do it in this review?

Reply:
can you open an issue for it to be addressed in the future?

Reply:
done